### DISCUSSION TITLE
Help needed to modify defaultpin

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#53] [Q&A] [answered]
I use "pin" feature to clip images from websites into my wiki. But if i clip multiple images from the same webpage using the pin feature, separate tiddler is created for each of the images even if its from the same webpage. Is it possible for the clipped  images from the same webpage to be saved in a single tiddler (just like the default snip feature for text)

**Comment by buggyj** [comment, marked as answer]
in the tiddler " defaultPin" , in the last column there in the term 'inc' - you can change this to 'append'

**Comment by arunnbabu81** [comment]
Thank you. Thanks for making this plug in and helping us.
